CHANGELOG.md

```diff
@@ -46,6 +46,8 @@ Replacement `tsi1` indexes will be automatically generated on startup for shards
 1. [20409](https://github.com/influxdata/influxdb/pull/20409): Improve messages in DBRP API validation errors.
 1. [20489](https://github.com/influxdata/influxdb/pull/20489): Improve error message when opening BoltDB with unsupported file system options.
 1. [20490](https://github.com/influxdata/influxdb/pull/20490): Fix silent failure to register CLI args as required.
+1. [20527](https://github.com/influxdata/influxdb/pull/20527): Don't leak .tmp files while backing up shards.
+1. [20527](https://github.com/influxdata/influxdb/pull/20527): Allow backups to complete while a snapshot is in progress.
```

Comment:
Are these 2 lines an accurate summary of the expected impact?

Reply:
I think they are close enough.  Sorry for no squash/merge on the original commits.